docs/api/text/postag.rst

```diff
@@ -0,0 +1,164 @@
+
+.. plot::
+    :context: close-figs
+    :alt: PosTag plot with Penn Treebank tags
+
+    from yellowbrick.text import PosTagVisualizer
+
+
+    tagged_stanzas = [
+        [
+            [
+                ('Whose', 'JJ'),('woods', 'NNS'),('these', 'DT'),
+                ('are', 'VBP'),('I', 'PRP'),('think', 'VBP'),('I', 'PRP'),
+                ('know', 'VBP'),('.', '.')
+                ],
+            [
+                ('His', 'PRP$'),('house', 'NN'),('is', 'VBZ'),('in', 'IN'),
+                ('the', 'DT'),('village', 'NN'),('though', 'IN'),(';', ':'),
+                ('He', 'PRP'),('will', 'MD'),('not', 'RB'),('see', 'VB'),
+                ('me', 'PRP'),('stopping', 'VBG'), ('here', 'RB'),('To', 'TO'),
+                ('watch', 'VB'),('his', 'PRP$'),('woods', 'NNS'),('fill', 'VB'),
+                ('up', 'RP'),('with', 'IN'),('snow', 'NNS'),('.', '.')
+                ]
+            ],
+        [
+            [
+                ('My', 'PRP$'),('little', 'JJ'),('horse', 'NN'),('must', 'MD'),
+                ('think', 'VB'),('it', 'PRP'),('queer', 'JJR'),('To', 'TO'),
+                ('stop', 'VB'),('without', 'IN'),('a', 'DT'),('farmhouse', 'NN'),
+                ('near', 'IN'),('Between', 'NNP'),('the', 'DT'),('woods', 'NNS'),
+                ('and', 'CC'),('frozen', 'JJ'),('lake', 'VB'),('The', 'DT'),
+                ('darkest', 'JJS'),('evening', 'NN'),('of', 'IN'),('the', 'DT'),
+                ('year', 'NN'),('.', '.')
+                ]
+            ],
+        [
+            [  
+                ('He', 'PRP'),('gives', 'VBZ'),('his', 'PRP$'),('harness', 'NN'),
+                ('bells', 'VBZ'),('a', 'DT'),('shake', 'NN'),('To', 'TO'),
+                ('ask', 'VB'),('if', 'IN'),('there', 'EX'),('is', 'VBZ'),
+                ('some', 'DT'),('mistake', 'NN'),('.', '.')
+                ],
+            [
+                ('The', 'DT'),('only', 'JJ'),('other', 'JJ'),('sound', 'NN'),
+                ('’', 'NNP'),('s', 'VBZ'),('the', 'DT'),('sweep', 'NN'), 
+                ('Of', 'IN'),('easy', 'JJ'),('wind', 'NN'),('and', 'CC'),
+                ('downy', 'JJ'),('flake', 'NN'),('.', '.')
+                ]
+            ],
+        [
+            [
+                ('The', 'DT'),('woods', 'NNS'),('are', 'VBP'),('lovely', 'RB'),
+                (',', ','),('dark', 'JJ'),('and', 'CC'),('deep', 'JJ'),(',', ','),
+                ('But', 'CC'),('I', 'PRP'),('have', 'VBP'),('promises', 'NNS'),
+                ('to', 'TO'),('keep', 'VB'),(',', ','),('And', 'CC'),('miles', 'NNS'),
+                ('to', 'TO'),('go', 'VB'),('before', 'IN'),('I', 'PRP'),
+                ('sleep', 'VBP'),(',', ','),('And', 'CC'),('miles', 'NNS'),
+                ('to', 'TO'),('go', 'VB'),('before', 'IN'),('I', 'PRP'),
+                ('sleep', 'VBP'),('.', '.')
+                ]
+        ]
+    ]
```

Comment:
:+1: I've added a SpaCy example.

Reply:
Ah, sorry for the confusion - but thanks for adding the universal tags example! I've realized I was a little unclear by what I meant; I was referring to the explicit creation of the list of list of token, tag tuples as reference data to what's expected by the visualizer. The secondary examples I was referring to were more something like:

```python
.. code-block:: python

    import nltk

    text = [
        "The woods are lovely, dark and deep,",
        "But I have promises to keep,", 
        "And miles to go before I sleep,", 
        "And miles to go before I sleep".,
    ]

    # Tokenize and tag text
    tagged_stanzas = [nltk.pos_tag(nltk.word_tokenize(sent)) for sent in text]

    # Create the visualizer 
    viz = PostagVisualizer()
    viz.fit(tagged_stanzas)
    viz.poof()
```

So I guess what I was saying was that I do really like the hardcoded example to show what kind of data was expected, but perhaps afterward we could create some code snippets to show how this is probably used in real-life, e.g. directly on strings with nltk and spacy -- but not in `.. plot::` directive so that the example code didn't execute and require extra dependencies for our documentation. 

Just wanted to clarify my comments - I'll leave it to you if you want to leave it like it is, that's perfectly fine!

Reply:
Ah, sorry for the confusion - but thanks for adding the universal tags example! I've realized I was a little unclear by what I meant; I was referring to the explicit creation of the list of list of token, tag tuples as reference data to what's expected by the visualizer. The secondary examples I was referring to were more something like:

```python
.. code-block:: python

    import nltk

    text = [
        "The woods are lovely, dark and deep,",
        "But I have promises to keep,", 
        "And miles to go before I sleep,", 
        "And miles to go before I sleep".,
    ]

    # Tokenize and tag text
    tagged_stanzas = [nltk.pos_tag(nltk.word_tokenize(sent)) for sent in text]

    # Create the visualizer 
    viz = PostagVisualizer()
    viz.fit(tagged_stanzas)
    viz.poof()
```

So I guess what I was saying was that I do really like the hardcoded example to show what kind of data was expected, but perhaps afterward we could create some code snippets to show how this is probably used in real-life, e.g. directly on strings with nltk and spacy -- but not in `.. plot::` directive so that the example code didn't execute and require extra dependencies for our documentation. 

Just wanted to clarify my comments - I'll leave it to you if you want to leave it like it is, that's perfectly fine!

Reply:
I'm sorry, I misunderstood what you were asking for. I'll add this to the follow-up task?

Reply:
It's completely my fault! And that would be great!